test/autoCleanup.noAfterEach.test.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from 'react'
+
+// This verifies that if RHTL_SKIP_AUTO_CLEANUP is set
+// then we DON'T auto-wire up the afterEach for folks
+describe('skip auto cleanup (no afterEach) tests', () => {
+  let cleanupCalled = false
+  let renderHook
+
+  beforeAll(() => {
+    afterEach = false
+    renderHook = require('src').renderHook
+  })
+
+  test('first', () => {
+    const hookWithCleanup = () => {
+      useEffect(() => {
+        return () => {
+          cleanupCalled = true
+        }
+      })
+    }
+    renderHook(() => hookWithCleanup())
+  })
+
+  test('second', () => {
+    expect(cleanupCalled).toBe(false)
```

Comment:
❓ Is this test supposed to check `true`, `expect(cleanupCalled).toBe(true)`?

It appears to be a copy of the above test, `autoCleanup.disabled.test.js`, but without the `process.env` setup.

You may need to clear the `require` cache and reset the `process.env` before running?

Reply:
No.  In the `beforeAll` am overriding `afterEach` to not be a function to test the case where the testing framework does not support it.  I was very worried about this, but I tested thoroughly and `jest` brings it back for the next test.

Happy to hear ideas for better ways to test this case.

Reply:
welp `jest` is pretty damn cool.